starknet_py/tests/e2e/docs/account_creation/test_deploy_prefunded_account.py

```diff
@@ -43,26 +49,30 @@ async def test_deploy_prefunded_account(
     await res.wait_for_acceptance()
     # docs: start
 
-    # Create an Account instance
-    account = Account(
-        address=address,
-        client=GatewayClient(net=network),
-        key_pair=key_pair,
-        chain=StarknetChainId.TESTNET,
-    )
+    # Define the client to be used to interact with StarkNet
+    client = GatewayClient(net=TESTNET)
+    chain = StarknetChainId.TESTNET
+    # docs: end
+
+    client = gateway_client
+    chain = chain_from_network(net=network, chain=StarknetChainId.TESTNET)
+    # docs: start
 
-    # Create and sign DeployAccount transaction
-    deploy_account_tx = await account.sign_deploy_account_transaction(
+    # Use `Account.deploy_account` static method to deploy an account
+    account_deployment_result = await Account.deploy_account(
+        address=address,
         class_hash=class_hash,
-        contract_address_salt=salt,
+        salt=salt,
+        key_pair=key_pair,
+        client=client,
+        chain=chain,
         constructor_calldata=[key_pair.public_key],
         max_fee=int(1e15),
     )
+    # Wait for deployment transaction to be accepted
+    await account_deployment_result.wait_for_acceptance()
 
-    resp = await account.client.deploy_account(transaction=deploy_account_tx)
-    await account.client.wait_for_tx(resp.transaction_hash)
-
-    # Since this moment account can be used to sign other transactions
+    # Since this moment account can be used as usual
```

Comment:
```suggestion
    # From now on, account can be used as usual
```
Since this moment sounds weird to me, but maybe it's just me 🤷